plugins/inputs/logparser/grok/influx_patterns.go

```diff
@@ -53,7 +53,7 @@ RESPONSE_TIME %{DURATION:response_time_ns:duration}
 EXAMPLE_LOG \[%{HTTPDATE:ts:ts-httpd}\] %{NUMBER:myfloat:float} %{RESPONSE_CODE} %{IPORHOST:clientip} %{RESPONSE_TIME}
 
 # Wider-ranging username matching vs. logstash built-in %{USER}
-NGUSERNAME [a-zA-Z\.\@\-\+_%]+
+NGUSERNAME [a-zA-Z0-9\.\@\-\+_%]+
```

Comment:
I've changed the COMMON LOG FORMAT pattern to match NOTSPACE for ident and auth. I wasn't able to find any information regarding character limitations on these. It appears that even whitespace is allowed based on #1810, but hopefully it's not very common.